### ISSUE TITLE
hi，i want to ask some  question

### ISSUE THREAD

**Issue by wudizuixiaosa** [JarrentWu1031/CCPL#4]
First of all, I'm glad to see your amazing works,，Can your model apply to the conversion of the style of underwater images to that of land? If I want to do so, what do I need to prepare? For example, I now have a ready-made underwater fish data set

**Comment by JarrentWu1031** [comment]
Thanks for your appreciation. I wonder if you mean style transfer or image-to-image translation. For the former, you could just download the pre-trained model and do the testing. For the latter, you may have to use some kind of image-to-image translation model and add CCPL during training.

**Comment by wudizuixiaosa** [comment]
Thank you for your timely reply. For example, cyclegan can complete the conversion from day to night, so I think CCPL can also complete the conversion from underwater to land. I wonder if what I said is clear? I have seen your general transformation from actual images to artistic styles, which is not what I said. So I want to ask if I want to complete the situation like I said, should I retrain?

**Comment by heraclitusj** [comment]
You're welcome. Here's the thing. In this paper, we propose CCPL and SCTNet. SCTNet is designed specially for style transfer, which means the broader image-to-image translation tasks might go beyond its power. However, the generic loss (CCPL) we proposed in the paper, is proved to be useful in image-to-image tasks. For example, we show horse-to-zebra results when adding CCPL on top of CUT model. You could also add CCPL on CycleGAN during training if you're interested.&nbsp;
 



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月1日(星期一) 下午4:04
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: Re: [JarrentWu1031/CCPL] hi，i want to ask some  question (Issue #4)





 
Thank you for your timely reply. For example, cyclegan can complete the conversion from day to night, so I think CCPL can also complete the conversion from underwater to land. I wonder if what I said is clear? I have seen your general transformation from actual images to artistic styles, which is not what I said. So I want to ask if I want to complete the situation like I said, should I retrain?

**Comment by wudizuixiaosa** [comment]
Thank you very much! I will try it soon
